spotguide/spotguide.go

```diff
@@ -51,6 +51,11 @@ const DeployApplicationStep = "deploy_application"
 
 var ctx = context.Background()
 
+func init() {
```

Comment:
Better solution would be calling this somewhere in main, at least indirectly, but I can live with it as a temporary solution if it would require a huge amount of work now. Even then a big, fat TODO warning that it should be refactored at some point.